ShNorah/safe-courier-rest-apis
# create-endpoints

**What does this PR do?**
create end-points
**Description of the task to be completed.**
creates safe-courier endpoints using a data structure
**How should this be manually tested?**

- clone this repository
- check out to develop branch
- run your server on localhost

**Any background context you want to provide?**
can't get parcels of specific user
**Screenshots(if appropriate)**
none
**Questions**
none
